docs/quickstart/cross-chain-transfers.md

```diff
@@ -0,0 +1,286 @@
+
+Copy the [network setting from above](#network-setting) into `evm/buildImportTx-xchain.ts`.
+
+Navigate to this part of the code and ensure that the `cHexAddress`(_Your C-Chain wallet address_) and `private key` are correct:
+
+```ts
+const cHexAddress: string = "<YOUR-CCHAIN-WALLET-ADDRESS-HERE>";
+const privKey: string = "<YOUR-PRIVATE-KEY-HERE>";
+```
+
+Run the import script:
+
+```sh
+$ ts-node examples/evm/buildImportTx-xchain.ts
+```
+
+This returns:
+
+```sh
+Success! TXID: ft33QHCH542wWUwrjdVf9APtAjPWqzqbqE43nCRECdbzhzVub
+```
+
+That's it! You've transferred AVAX from the X-Chain to C-Chain!
+
+You can verify this TX by copy / pasting the import TXID into [Avascan](https://testnet.avascan.info/blockchain/c/tx/ft33QHCH542wWUwrjdVf9APtAjPWqzqbqE43nCRECdbzhzVub)
+
+### Transfer from the C-Chain to the X-Chain
+
+To return the AVAX back to the X-Chain, you need to do the transfer in the opposite direction.
+
+#### Export the Avax Token From C-Chain to X-Chain
+
+Select the [**`examples/evm`**](https://github.com/ava-labs/avalanchejs/tree/master/examples/evm) folder to view the AvalancheJS C-Chain examples. To export AVAX from the X-Chain to the C-Chain, select [`evm/buildExportTx-xchain-avax.ts`](https://github.com/ava-labs/avalanchejs/blob/master/examples/evm/buildExportTx-xchain-avax.ts)
+
+Make necessary changes as above for private key and network settings.
+
+You can change the amount of AVAX to send by editing the _BN_ variable: `avaxAmount`. The sample code assigns this as `1e7` or `10000000` (0.01 AVAX)
+
+The fee here will only be for exporting the asset. The import fees will be deducted from the UTXOs present on the Exported Atomic Memory, a memory location where UTXOs stay after getting exported but before being imported.
+
+```ts
+let avaxAmount: BN = new BN(1e7);
+let fee: BN = baseFee.div(new BN(1e9));
+fee = fee.add(new BN(1e6));
+```
+
+:::tip
+You do not need to adjust `fee`, you will however adjust `avaxAmount`, the variable that the sample code uses to define the amount of AVAX that the user will send. Be sure to send enough to support import fees (constant .001 AVAX). Sending less than `1e6` or `1000000` (0.001 AVAX) may cause the import txn to fail.
+:::
+
+Run the export script:
+
+```zsh
+avalanchejs $ ts-node examples/evm/buildExportTx-xchain-avax.ts
+Success! TXID: UAez3DTv26qmhKKFDvmQTayaXTPAVahHenDKe6xnUMhJbKuxc
+```
+
+#### Import the Avax Token From C-Chain to X-Chain
+Before we run the example import script, we need to make some changes to the code:
+
+1. Change the [Network Setting](./cross-chain-transfers.md#network-setting) to meet [Local](./cross-chain-transfers.md#local-workflow), [Fuji](./cross-chain-transfers.md#network-setting), or [Mainnet](./cross-chain-transfers.md#mainnet-workflow) network requirements.
```

Comment:
why do you mention Mainnet here?

Reply:
Also `#network-setting` is enough for the linking.

Reply:
Please this whole section is under `Fuji Workflow`. Please remove all references to Local and Mainnet here.